cmake/cmake_tools.cmake

```diff
@@ -0,0 +1,187 @@
+#
+# Copyright (C) 2018 by George Cave - gcave@stablecoder.ca
```

Comment:
Should this file be copied here? Would there be a way to use the upstream file (`ExternalProject_Add(..)` or something similar)?

Reply:
Since I plan to build everything on this package I would prefer to copy versus other options.